src/core/trackingmodel.h

```diff
@@ -0,0 +1,77 @@
+/***************************************************************************
+ trackingmodel.h - TrackingModel
+
+ ---------------------
+ begin                : 20.02.2020
+ copyright            : (C) 2020 by David Signer
+ email                : david (at) opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef TRACKINGMODEL_H
+#define TRACKINGMODEL_H
+
+#include <QAbstractItemModel>
+
+#include "rubberbandmodel.h"
+#include "tracker.h"
+
+class RubberbandModel;
+class Track;
+
+class TrackingModel : public QAbstractItemModel
+{
+    Q_OBJECT
+
+  public:
+    explicit TrackingModel( QObject *parent = nullptr );
+    ~TrackingModel() override;
+
+    enum TrackingRoles
+    {
+      DisplayString = Qt::UserRole,
+      VectorLayer,
+      RubberModel,
+      TimeInterval,
+      MinimumDistance,
+      Conjunction,
```

Comment:
```suggestion
      Conjunction,
```

Can you add a docstring to at least this role?